### ISSUE TITLE
Bug - Select - VO cannot navigate/select checkbox variants

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-react#7534]
**Describe the problem**
Follow up based on conversation found at https://github.com/patternfly/patternfly-react/pull/7505#pullrequestreview-1001495795

When the options for a Select component have checkboxes, VO is unable to properly navigate between or select options. See video below (this issue occurs even without the view more or grouped variation despite the name of the video):

https://user-images.githubusercontent.com/70952936/172920603-aca84220-196c-4b1c-b1bd-8245fcc8837d.mov

**How do you reproduce the problem?**
1. Go to [Select checkbox input example](https://www.patternfly.org/v4/components/select#checkbox-input)
2. Turn on VO, navigate to and open the toggle
3. Try navigating via VO key + right arrow key, and notice that focus isn't placed properly + selecting an option doesn't work properly

**Expected behavior**
VO should be able to navigate between each checkbox, and the correct checkbox should be selected upon key press

**Comment by thatblindgeye** [comment]
Based on some discussion and research into this issue, the conclusion that was come to is that the most viable solution is our planned composable select as part of the epic at https://github.com/patternfly/patternfly-react/issues/6014 (the current [Composable checkbox select](https://www.patternfly.org/v4/demos/composable-menu#composable-simple-checkbox-select) demo works as intended).

While some attempts were close, including combinations of wrapping a checkbox select variant's options in an `<li>` and passing in a `role=option`, this would either cause other issues to be flagged by aXe, or would require further modifications to the current Select implementation (one immediate thing would be having to create additional conditionals to take into account Select variants that are both `isGrouped` and `checkbox`, to avoid the default `<li>` styling).

Currently the checkbox variant can be navigated via VoiceOver via the Tab key.

Based on this, I will be closing this issue and its linked PR.
